site/content/en/_index.md

```diff
@@ -16,6 +16,17 @@ apply Kubernetes configuration.
 
 {{% /blocks/lead %}}
 
+{{< blocks/section color="light" >}}
```

Comment:
What is the motivation behind this change? Can you attach a screenshot as it how it looks like after the change? Do you think the installation icon is not at right place?

Reply:
Having the installation link clearly at the top makes it easier for newcomers to take the first step and install kpt. It also brings us more in line with other websites like docs.docker.com, nodejs.org, helm.sh which all have installation directions first, then guides/reference. Also, the install icon was broken and I fixed it.
Before: 
<img width="295" alt="Screen Shot 2020-06-29 at 11 13 34 AM" src="https://user-images.githubusercontent.com/2899196/86023948-39fec700-b9e1-11ea-97ba-a0b28960e9e2.png">
After:
<img width="283" alt="Screen Shot 2020-06-29 at 11 14 30 AM" src="https://user-images.githubusercontent.com/2899196/86023966-3f5c1180-b9e1-11ea-8fe0-d1cb21ff35d3.png">